index.js

```diff
@@ -54,35 +56,62 @@ app.post('/challenges', (req, res) => {
       db.User.create({
         email: req.body.participant.email
       }).then((participant) => {
-        console.log('patricipant is');
-        console.log(participant);
-        var mailOptions = {
-          from: '"Fred Foo 👥" <foo@blurdybloop.com>',
-          to: participant.email,
-          subject: 'Invitation to challenge',
-          text: ` Hello,
-          You has been invited to do a challenge, Please click this below link to see the details`
-          };
-        transporter.sendMail(mailOptions, function(error, info){
-            if(error){
-              return console.log(error);
-            }
-            console.log('Message sent: ' + info.response);
-          });
+        challenge.uuid = base64url(crypto.randomBytes(48));
```

Comment:
you can create a challenges route and put it there, along with the new task/new user form and challange display